modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Xref_update_conf.pm

```diff
@@ -116,6 +116,7 @@ sub pipeline_analyses {
              -flow_into  => { '2->A' => 'schedule_source',
                               'A->2' => 'schedule_dependent_source'},
              -rc_name    => 'small',
+             -meadow_type => 'LOCAL',
```

Comment:
I seem to remember an attempt at removing all LOCAL meadows from production pipelines, is this just a missing merge?

Reply:
I've only found out about recently that local meadows and Hive 2.5 don't go that well. I'll remove the commit!

Reply:
Removed the commit. Thank you for catching this in the review!